### DISCUSSION TITLE
Move some methods from the XXXConstants.java template

### DISCUSSION THREAD

**Discussion by vsajip** [javacc21/javacc21#197] [Ideas]
I'd like to move this and related methods to the lexer - any objections? The `Constants` interface isn't really a thing in the other generation languages. The other methods like `displayChar()`, `addEscapes()` etc. could move to a static utility class.

https://github.com/javacc21/javacc21/blob/e0dfb24ed7f64649564ced1feb196c53bee02786/src/ftl/java/Constants.java.ftl#L137

**Comment by revusky** [comment]
Well, I guess if you want to re-organize things like this somewhat, I have no real objection. As best I can see, it doesn't really matter very much.

**Comment by vsajip** [comment]
Now tracked in #205.
